value.go

```diff
@@ -48,7 +48,7 @@ import (
 
 // maxVlogFileSize is the maximum size of the vlog file which can be created. Vlog Offset is of
 // uint32, so limiting at max uint32.
-var maxVlogFileSize = math.MaxUint32
+var maxVlogFileSize = uint32(math.MaxUint32)
```

Comment:
That does make sense, untyped numeric variables are `int`, which is not big enough to hold `math.MaxUint32` on 32 bit architectures.